content/posts/2021-11-18-relicensing-update.md

```diff
@@ -0,0 +1,328 @@
+---
+author: Kristof Beyls
+date: "2021-11-18"
+tags: ["relicensing", "foundation"]
+title: LLVM relicensing update & call for help
+---
+
+In this blog post, I'd like to summarize the main points I talked about in the
+relicensing update presentation at the 2021 LLVM developers meeting.
```

Comment:
I'd suggest spelling this "LLVM Developer's Meeting"

Reply:
Thanks, I'll follow this suggestion